.github/component-owners.yml

```diff
@@ -0,0 +1,8 @@
+components:
+  /:
```

Comment:
`/` is used for all:
https://github.com/dyladan/component-owners/blob/main/src/utils.ts#L51